### PR TITLE
New package: tree-sitter-0.17.3

### PR DESCRIPTION
**Pull request by gbrlsnchs**

### CODE REVIEW

**Comment by gbrlsnchs** [comment]
This package will be needed in the future in order to build Neovim 0.5.0. :smile:

File: srcpkgs/tree-sitter/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'tree-sitter'
+pkgname=tree-sitter
+version=0.17.3
+revision=1
+build_style=gnu-makefile
+short_desc="Incremental parsing system for programming tools"
+maintainer="Gabriel Sanches <gabriel@gsr.dev>"
+license="MIT"
+homepage="https://tree-sitter.github.io/"
+_base_url="https://github.com/${pkgname}/${pkgname}"
```
Comment (Duncaen) [inline srcpkgs/tree-sitter/template:10]: I don't think the variable is necessary, having the full url in changelog without variables is always a lot easier to handle.

Comment (Duncaen) [reply srcpkgs/tree-sitter/template:10]: There is no changelog on the releases page, so listing this is giving a false impression.

Comment (gbrlsnchs) [reply srcpkgs/tree-sitter/template:10]: Sorry, I got a little confused. To which change log are you referring?

Comment (gbrlsnchs) [reply srcpkgs/tree-sitter/template:10]: Nevermind, I just realized there is a `changelog` property. I'll fix this, thanks for the tip!

Comment (Duncaen) [reply srcpkgs/tree-sitter/template:10]: Yes remove the `_base_url` variable and just use the full url and remove the changelog variable as it doesn't provide any useful information.

Comment (gbrlsnchs) [reply srcpkgs/tree-sitter/template]: Done,  is it OK the way it is?

**Comment by ericonr** [comment]
Merged in 79df2fe596fccd566b1a83d79b2a0bf89a437698

**Comment by gbrlsnchs** [comment]
Great news, thanks @ericonr!

**Pull request closed without merging**